java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/model/Version.java

```diff
@@ -0,0 +1,28 @@
+package com.google.cloud.model;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+public class Version {
```

Comment:
Would you add javadoc? I guess this corresponds to the concept in deps.dev. If so, please add the reference.

Reply:
Done.